pom.xml

```diff
@@ -93,6 +93,8 @@ LICENSE file.
     <tarantool.version>1.6.5</tarantool.version>
     <aerospike.version>3.1.2</aerospike.version>
     <solr.version>5.4.0</solr.version>
+    <rados.version>0.3.0-SNAPSHOT</rados.version>
+    <json.version>20160212</json.version>
```

Comment:
this should just be set in the pom for the rados client, since it needn't be shared across different datastore bindings..